packages/params/src/index.ts

```diff
@@ -255,11 +255,11 @@ export const BLOB_TX_TYPE = 0x03;
 export const VERSIONED_HASH_VERSION_KZG = 0x01;
 
 // ssz.deneb.BeaconBlockBody.getPathInfo(['blobKzgCommitments',0]).gindex
-export const KZG_COMMITMENT_GINDEX0 = ACTIVE_PRESET === PresetName.minimal ? 864 : 221184;
+export const KZG_COMMITMENT_GINDEX0 = ACTIVE_PRESET === PresetName.minimal ? 1728 : 221184;
```

Comment:
this changed for electra or was the value incorrect? then gindex0 constant needs to be forkaware

Reply:
in the spec PR the minimal preset values for deneb were changed [here](https://github.com/ethereum/consensus-specs/pull/4023/files#diff-105bcf56c68ccee23f29448d8cad7a9376736382b9ab9b39de28fcadd35625c0R8), I thought you have to add new constants for that but apparently not

but agree the gindex should be fork aware

Reply:
then this will break the current deneb minimal devnets

Reply:
ahh because we retrospectively changed deneb max vals, sounds very random to me to do (in spec)